daemon/buildinfo.c

```diff
@@ -237,41 +236,42 @@ struct install_type_info get_install_type() {
 
         while ((s = fgets_trim_len(buf, 256, fp, &len))) {
             if (!strncmp(buf, "INSTALL_TYPE='", 14))
-                ret.install_type = strdupz((char *)get_value_from_key(buf, "INSTALL_TYPE"));
+                *install_type = strdupz((char *)get_value_from_key(buf, "INSTALL_TYPE"));
             else if (!strncmp(buf, "PREBUILT_ARCH='", 15))
-                ret.prebuilt_arch = strdupz((char *)get_value_from_key(buf, "PREBUILT_ARCH"));
+                *prebuilt_arch = strdupz((char *)get_value_from_key(buf, "PREBUILT_ARCH"));
             else if (!strncmp(buf, "PREBUILT_DISTRO='", 17))
-                ret.prebuilt_distro = strdupz((char *)get_value_from_key(buf, "PREBUILT_DISTRO"));
+                *prebuilt_dist = strdupz((char *)get_value_from_key(buf, "PREBUILT_DISTRO"));
         }
         fclose(fp);
     }
     freez(install_type_filename);
-
-    return ret;
 }
 
 void print_build_info(void) {
-    struct install_type_info t = get_install_type();
+    char *install_type = NULL;
+    char *prebuilt_arch = NULL;
+    char *prebuilt_distro = NULL;
+    get_install_type(&install_type, &prebuilt_arch, &prebuilt_distro);
 
     printf("Configure options: %s\n", CONFIGURE_COMMAND);
 
-    if (t.install_type == NULL) {
+    if (install_type == NULL) {
         printf("Install type: unknown\n");
     } else {
-        printf("Install type: %s\n", t.install_type);
+        printf("Install type: %s\n", install_type);
     }
 
-    if (t.prebuilt_arch != NULL) {
-        printf("    Binary architecture: %s\n", t.prebuilt_arch);
+    if (prebuilt_arch != NULL) {
+        printf("    Binary architecture: %s\n", prebuilt_arch);
```

Comment:
Reeeeeally picky this one, but `Binary architecture:` and `Packaging distro:` have some spaces in front, while `Install type:` doesn't :-)

So we print:
```
evas@winterland ~/stuff/netdata/netdata ↪ /home/evas/stuff/netdata/install_netdata_timo_inst_type/netdata/usr/sbin/netdata -W buildinfo                        
Version: netdata v1.33.0-2-g4472d39bd
Configure options:  '--prefix=/home/evas/stuff/netdata/install_netdata_timo_inst_type/netdata/usr' '--sysconfdir=/home/evas/stuff/netdata/install_netdata_timo_inst_type/netdata/etc' '--localstatedir=/home/evas/stuff/netdata/install_netdata_timo_inst_type/netdata/var' '--libexecdir=/home/evas/stuff/netdata/install_netdata_timo_inst_type/netdata/usr/libexec' '--libdir=/home/evas/stuff/netdata/install_netdata_timo_inst_type/netdata/usr/lib' '--with-zlib' '--with-math' '--with-user=netdata' '--disable-lto' '--with-bundled-protobuf' 'CFLAGS=-O1 -ggdb -Wall -Wextra -fsanitize=address -static-libasan -fno-omit-frame-pointer -Wformat-signedness -fstack-protector-all -DNETDATA_INTERNAL_CHECKS=1 -D_FORTIFY_SOURCE=2 -DNETDATA_VERIFY_LOCKS=1' 'LDFLAGS='
Install type: hello
    Binary architecture: tada
    Packaging distro: Gentoo rules
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
```

Really not important, just aesthetic (unless we want it like that!)

Reply:
Otherwise, tested it, checked posthog, buildinfo, api/v1/info, seems ok also with the file missing, etc.!

Reply:
This is intentional, we want those grouped under the install type because they’re supplementary information.